src/components/editor/resourceURIMessage.jsx

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 Stanford University see LICENSE for license
+
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+// Renders the resource URI message after Save & Publish
+const ResourceURIMessage = (props) => {
+  if (!props.show) {
+    return null
+  }
+
+  return (
+    <div>
+      <h4>URI for this resource: &lt;{ props.uri }&gt;</h4>
```

Comment:
should there be an if in here to be sure props.uri has a value?

should there be simple unit tests that check that the uri is output when present, and that it outputs something suitable when it's not present?  (first part only, if having a check in here isn't warranted.)

lastly, should there be an "integration" test showing this message displays after Save & Publish

Reply:
i think a simple `ResourceURIMessage.WrappedComponent` test for the basic cases (no show, show but no URI, show is true and there's a URI) would be easy and a nice start (shouldn't require any mocking).

agreed an integration test that shows things are wired up correctly via `mapStateToProps` would be nice too.